ethereum/Deploying.md

```diff
@@ -1,61 +1,109 @@
 # Deploying Contracts to Production
 
-Running the Truffle migrations in [`migrations/prod`](migrations/prod) or [`migrations/prod-receiver`](migrations/prod-receiver/) will deploy the contracts to production. The `prod-receiver` migrations should be used when you need to deploy to a chain that is unsupported by the Wormhole network. The Wormhole Receiver contract acts as a read-only Wormhole endpoint that can verify Wormhole messages even if the Wormhole network has not yet connected the chain.
+## EVM network configurations
+
+Each network that Pyth is deployed on has some configurations stored on this repo as described below:
+
+1. [`truffle-config.js`](./truffle-config.js) contains list of different networks with their configurations that includes:
+   - `provider`: It is the network provider which is an HDWalletProvider with mnemonic stored on $MNEMONIC
```

Comment:
suggest starting with "The network provider" . "it is" is redundant

"stored in the $MNEMONIC environment variable"